src/ztc/datamodel/models/resultaattype.py

```diff
@@ -6,8 +6,9 @@
 import requests
 from zds_schema.constants import (
     Archiefnominatie,
-    BrondatumArchiefprocedureAfleidingswijze as Afleidingswijze, ObjectTypes
+    ZaakobjectTypes as Afleidingswijze, ObjectTypes
```

Comment:
that's a broken import - the alias should point to `BrondatumArchiefprocedureAfleidingswijze`, and `ObjectTypes` should be replaced with `ZaakobjectTypes`

Reply:
fixed it